### ISSUE TITLE
Review site accessibility

### ISSUE THREAD

**Issue by katjam** [GenderedIntelligence/the-trans-dimension#39]
## Description
Run down accessibility checklist to make sure we're doing all the right things. Obviously we will be accessibility first as we build but it's worthwhile to do an audit after we have built a few pages.

### Must do: 
- [x] Use a WAVE tool to evaluate each page https://wave.webaim.org/
- [x] Run through site use with keyboard only
- [x] Run through site use with screenreader
- [x] Check all meaningful images for alt text
- [x] Check all form inputs for labels
- [x] Check all links for meaningful & unique link text
- [x] Check all headings are correct for thier semantic meaning

### Should do:
- [x]  Run down https://www.w3.org/WAI/WCAG21/quickref and ticket anything that does not meet AA

## Notes
- Starling bank has a nice statement https://www.starlingbank.com/website-accessibility. We ought to be doing all the things listed there as "What we are doing"

**Comment by katjam** [comment]
@jasminechatfield We can do the "Must do" as individual tickets but I found it useful to list them in one place.

**Comment by ivankocienski** [comment]
Might be worth watching [this talk](https://www.youtube.com/watch?v=G_TNRuoF3FE) ?

**Comment by katjam** [comment]
I mean, having said that, if you are interested in how types might be used to enforce accessibility then yes, watch. It makes some interesting observations and was a good endeavour. I don't think the library has been widely adopted though - maybe because developers have to ultimately know how to write the accessible markup whether they use a tool to enforce it or not.

**Comment by katjam** [comment]
This is a pretty good overview / starting point https://developer.mozilla.org/en-US/docs/Learn/Accessibility

And if you want some videos - then these are linnked from there: https://egghead.io/courses/start-building-accessible-web-applications-today

I learned most of my fundamentals from working with Heydon in the early days of Neontribe, it was just me and him. He's pretty opinionated but has some good resources like: https://inclusivedesignprinciples.org/

**Comment by katjam** [comment]
OK. I do think it is fundamentally important though and that we ought to be doing everything with an accessible 1st attitude. It's not that big of a job if you pay attention to it as you go. We can work on this for future.

**Comment by kimadactyl** [comment]
I agree, just trying to be pragmatic here. I trust you to use good accessible code as it is. I think a lot of this stuff is really nice to have but isn't going to stop anyone using the site so I see this as probably iteration 2 already.

**Comment by erbridge** [comment]
I keep spotting accessibility issues in the code, so I've pulled this back into the public release to do a high level pass of the site. I think the action is for someone who already knows about this stuff to pull out more issues for each point (or quickly fix them).

**Comment by katjam** [comment]
Agree. I can take this one back and make the tickets as per the original
plan. Once they are articulated we can decide on priority.

**Comment by aaaaargZombies** [comment]
Ivans report https://www.notion.so/gfsc/Trans-Dimension-Usability-Review-541e29bd0b124cbeaffb39b880dce51a

**Comment by katjam** [comment]
@ivan-kocienski-gfsc can you please tick the items in must do that you completed? 

Then, convert the outstanding tasks to issues - on the backlog.

After that, we can maybe close this?
